tests/build.gradle

```diff
@@ -46,6 +48,7 @@ dependencies {
     compile 'io.spray:spray-testkit_2.11:1.3.3'
     compile 'com.google.code.gson:gson:2.3.1'
     compile 'org.scalamock:scalamock-scalatest-support_2.11:3.4.2'
+    testCompile 'com.typesafe.play:play-ws_2.11:2.5.11'
```

Comment:
Make it `compile`. I chose to have those as compile-time dependencies, because we have an anomalie having a seperate test-project anyway. Gradle doesn't import testCompile dependencies if i reference the test project (which catalog etc. do for example).